chef/cookbooks/hyperv/recipes/register_services_ceilometer.rb

```diff
@@ -0,0 +1,21 @@
+raise if not node[:platform] == 'windows'
+
+powershell "register_services" do
+  code <<-EOH
+    if (-not (Get-Service "#{node[:service][:ceilometer][:name]}" -ErrorAction SilentlyContinue))
+    {
+      # $secpasswd = ConvertTo-SecureString "crowbar" -AsPlainText -Force
+      # $credentials = New-Object System.Management.Automation.PSCredential (".\\Administrator", $secpasswd)
```

Comment:
Any reason to keep these comments? Especially as we have the password configurable now, which make them wrong.